[terra-alert] Update alert notification focus shift for dismiss (#3823)
cerner/terra-core: packages/terra-alert/CHANGELOG.md

## Unreleased

* Changed

>   * Updated focus shift to rendered `alert` notification banners content that include an interactable action element and/or dismiss.

Reviewer: Can we reword it to something like this?
```suggestion
  * Updated focus shift for `alert` notification banners to include notification banners that have an interactable action element and/or dismiss.
```

Author: Updated changelog here https://github.com/cerner/terra-core/pull/3823/commits/d4700b8f71c7da3a79709a02b3651a0419dcdd71#diff-b62ab7ae03f56c47f8aba033b108058b187c6bb44ff7dbe93a72c454c1393279

## Unreleased

* Added
  * Added focus shift to rendered `alert` notification banners content that include an interactable action element and/or dismiss.

* Changed

>   * Locked `uuid` to `8.2.0` to maintain IE compatibility.

Reviewer 2: This change was already released so can we rebase with main so this PR is up to date?

Author: Rebased changes with main and updated changelog here https://github.com/cerner/terra-core/pull/3823/files#diff-b62ab7ae03f56c47f8aba033b108058b187c6bb44ff7dbe93a72c454c1393279
